FluentScheduler.UnitTests/Scheduler/ScheduleTests.cs

```diff
@@ -262,5 +306,25 @@ public void UseUtcAfterStop()
             Assert.AreEqual(expectedNow.Hour, resultedNow.Hour);
             Assert.AreEqual(expectedNow.Minute, resultedNow.Minute);
         }
+
+        [TestMethod]
+        public void WaitForCancellation()
+        {
+            // Arrange
+            var cancelled = false;
+            var schedule = new Schedule(async (cancellationToken) =>
+            {
+                cancellationToken.WaitHandle.WaitOne(1000);
+                cancelled = cancellationToken.IsCancellationRequested;
```

Comment:
Minor thing: I believe `cancelled = cancellationToken.WaitHandle.WaitOne(1000)` would do the same.

Reply:
Fixed.